ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesImageSinglePage.tsx

```diff
@@ -1,9 +1,205 @@
+                            className="pf-u-danger-color-100"
+                            icon={ExclamationCircleIcon}
+                        />
+                        <Title headingLevel="h2">{getAxiosErrorMessage(error)}</Title>
+                    </EmptyState>
+                </Bullseye>
+            </PageSection>
+        );
+    } else {
+        mainContent = (
+            <>
+                <PageSection variant="light">
+                    {imageData ? (
+                        <Flex direction={{ default: 'column' }}>
+                            <Title headingLevel="h1" className="pf-u-mb-sm">
+                                {imageName}
+                            </Title>
+                            <ImageDetailBadges imageData={imageData} />
+                        </Flex>
+                    ) : (
+                        <Flex
+                            direction={{ default: 'column' }}
+                            spaceItems={{ default: 'spaceItemsXs' }}
+                            className="pf-u-w-50"
+                        >
+                            <Skeleton fontSize="2xl" />
+                            <Skeleton fontSize="sm" />
+                        </Flex>
+                    )}
+                </PageSection>
+                <PageSection variant="light" padding={{ default: 'noPadding' }}>
+                    <Tabs
+                        activeKey={activeTabKey}
+                        onSelect={handleTabClick}
+                        component={TabsComponent.nav}
+                        className="pf-u-pl-md"
+                        mountOnEnter
+                        unmountOnExit
+                    >
+                        <Tab
+                            eventKey="Vulnerabilities"
+                            title={<TabTitleText>Vulnerabilities</TabTitleText>}
+                        >
+                            <ImageSingleVulnerabilities />
+                        </Tab>
+                        <Tab
+                            eventKey="Resources"
+                            title={<TabTitleText>Resources</TabTitleText>}
+                            isDisabled
+                        >
+                            <ImageSingleResources />
+                        </Tab>
+                    </Tabs>
+                </PageSection>
+            </>
+        );
+    }
+
+    return (
+        <>
+            <PageSection variant="light" className="pf-u-py-md">
+                <Breadcrumb>
+                    <BreadcrumbItemLink to={workloadCveOverviewImagePath}>
+                        Images
+                    </BreadcrumbItemLink>
+                    {!error && (
+                        <BreadcrumbItem isActive>
+                            {imageData ? (
+                                imageName
+                            ) : (
+                                <Skeleton screenreaderText="Loading image name" width="200px" />
```

Comment:
should we have the `screenreaderText` for the `Skeleton` components above as well?